src/main/java/mods/railcraft/common/util/inventory/IInventoryImplementor.java

```diff
@@ -23,6 +23,9 @@
 public interface IInventoryImplementor extends IInventory, IInventoryComposite {
     IInventory getInventory();
 
+    @Override
+    boolean isUsableByPlayer(EntityPlayer player);
```

Comment:
Redundant?

Reply:
This serves as a reminder that implementors must implement this method instead of delegating.

Reply:
Fair enough.

Reply:
Added documentation to clarify as well.